### PR TITLE
Screenreader Announce suggested terms

### PR DESCRIPTION
**Pull request by tyler-spangler6**
## Summary

- During the Staging Review for the Contention Classification team, it was identified that the suggested terms were not announced by the screen reader.  This ticket corrects this and enables the screen reader to announce results.  The below screen shot shows what the screen reader announces. 

## Related issue(s)

-  [Staging Review Ticket #84584](https://github.com/department-of-veterans-affairs/va.gov-team/issues/84584) 

## Testing done

- Manually testing completed and shown in screen shot above. 

## Screenshots
image
_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/components/ComboBox.jsx [hunk 1]
```diff
@@ -266,15 +273,16 @@ export class ComboBox extends React.Component {
...
         tabIndex={0}
         onMouseEnter={e => {
           this.handleMouseEnter(e, 0);
         }}
-        onKeyDown={() => null}
+        onKeyDown={this.handleKeyPress}
         label="new-condition-option"
         role="option"
-        aria-selected={searchTerm === option}
+        aria-selected="false"
```
Comment (lukey-luke) [inline src/applications/disability-benefits/all-claims/components/ComboBox.jsx:285]: FYI reviewer, we were experiencing an issue where Screen reader results for highlighted options were announced twice. Apparently this fixes that so screen reader will read the `<li>` once `¯\_(ツ)_/¯`

**Review by lukey-luke — APPROVED** [review]
Nice work Tyler! Looks good to me!

File: src/applications/disability-benefits/all-claims/components/ComboBox.jsx (hunk 1 above)
Comment (tblackwe) [reply src/applications/disability-benefits/all-claims/components/ComboBox.jsx:285]: Which `screenreader` does this error occur in? Is it a consistent behavior across multiple readers?

**Review by tblackwe — APPROVED** [review]

File: src/applications/disability-benefits/all-claims/components/ComboBox.jsx (hunk 1 above)
Comment (tblackwe) [reply src/applications/disability-benefits/all-claims/components/ComboBox.jsx:285]: nevermind, I see this came from a CAIA request. I am confident their solution is consistent.

**Review by tblackwe — APPROVED** [review]

**Pull request merged**